### PR TITLE
development

### PR DESCRIPTION
**Pull request by kyracq**
https://kyraa-step-2020.appspot.com/

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -56,6 +61,36 @@ main {
...
+  color: var(--green);
+}
+
+.color-buttons .blue i {
+  color: var(--blue);
+}
+
+.color-buttons .purple i {
+  color: var(--purple);
+}
```
Comment (shulijones) [inline portfolio/src/main/webapp/style.css:88]: I wonder if there's a way to do this that doesn't involve listing each one here - it doesn't seem very DRY. Would it make more sense to just set the color inside the HTML for each button in index.html?

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -196,6 +273,42 @@ p {
...
   width: fit-content;
+}
+
+.btn:hover {
+  cursor: pointer;
+  filter: brightness(85%);
+
+}
+
+#back-to-top-btn {
```
Comment (shulijones) [inline portfolio/src/main/webapp/style.css:286]: This is a small design comment, but I feel like the back-to-top button could be larger or easier to see. I missed it the first couple times I scrolled through your site.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -33,54 +33,108 @@
 					<span class="link-text">Experience</span>
 				</a>
 			</li>
+			<div class="color-buttons">
```
Comment (shulijones) [inline portfolio/src/main/webapp/index.html]: This is all four spaces again. I think the cloud shell editor goes back to 4 whenever you reload it - I set it to 2 permanently by going into File > Settings > Open Preferences > Editor and changing tabSize.

**Review by shulijones — APPROVED** [review]
There's a couple different big changes here, so it's quite a bit of code. But it all looks great to me! The color change is super cute and the fades that you've put in work well.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (kyracq) [reply portfolio/src/main/webapp/index.html]: Thanks! Yeah I couldn't figure out how to make it permanently two spaces

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -86,3 +95,40 @@ function jobsBack() {
   setContent();
 }
 
+/**
+ * Change accent color to selected color
+ */
+function changeAccentColor(selectedButton) {
+  var classes = selectedButton.className.split(' ');
```
Comment (nsri-google) [inline portfolio/src/main/webapp/script.js]: use let or const instead of var 

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

**Review by otobin — APPROVED** [review]
LGTM! I'll echo Shuli's comment about the back to top button but other than that this looks really good.

**Pull request merged**